myproject/app/api/commant.py

```diff
@@ -7,52 +7,55 @@
 from flask_jwt_extended import JWTManager, create_access_token
 from app.models.model import db, user
 from flask_jwt_extended import get_jwt_identity, jwt_required
-
+from app.services.commant_services import target , comments
+from app.error_management.error_response import e_response , Response
+from app.error_management.success_response import success_response
+from app.validator.validators import check_command_required_fields
 
 bp = Blueprint('authe', __name__, url_prefix='/auth')
 @bp.route('/posts/<int:post_id>/comments', methods=['POST'])
 @jwt_required()
 def create_comment(post_id):
+    data = request.json
     current_user_id = get_jwt_identity()
```

Comment:
make meaningful name of variables,  if this is current_user_id then why you are again getting the id in next line